### PR TITLE
support HTML lang tag

### PR DESCRIPTION
**Pull request by casperdcl**
A `lang` attribute is [considered essential](https://www.w3schools.com/tags/ref_language_codes.asp)

### CODE REVIEW

**Comment by chrisrhymes** [comment]
Thanks for your pull request! Please can you change the below so there is a default lang for users that haven't set it in the _config.yml file. 

```html
{% if site.lang %}
 lang="{{ site.lang }}"{% endif %}
```

to 
```html
lang="{{ site.lang | default: 'en' }}"
```

**Comment by casperdcl** [comment]
Yeah I wasn't too sure about setting this default becsuse it may potentially change behaviour... e.g. a French site usually viewed on a browser configured to assume French by default will now be forced to assume English.

What do you think?

**Comment by chrisrhymes** [comment]
That's a very good point!

File: _layouts/default.html [hunk 1]
```diff
@@ -2,16 +2,18 @@
...
+    {% assign content_width = 'is-8' %}
+{% else %}
+    {% assign content_width = 'is-12' %}
 {% endif %}
 
 <!DOCTYPE html>
-<html {% if site.fixed_navbar %} class="has-navbar-fixed-{{ site.fixed_navbar }}" {% endif %}>
+<html {% if site.fixed_navbar %}
+ class="has-navbar-fixed-{{ site.fixed_navbar }}"{% endif %}{% if site.lang %}
+ lang="{{ site.lang }}"{% endif %}>
```
Comment (casperdcl) [inline _layouts/default.html:16]:
```suggestion
<html
 lang="{{ site.lang | default: 'en' }}"{% if site.fixed_navbar %}
 class="has-navbar-fixed-{{ site.fixed_navbar }}"{% endif %}>
```

maybe worth doing this and adding a note in the docs about setting `lang:` in `_config.yml`

**Pull request merged**